Modify REQ-4-1 to turn it into a startup error

ftrader opened the pull request.

deadalnix commented on BUIP-UAHF/buip-uahf-technical-spec.md:
```diff
@@ -75,23 +75,17 @@ the risk of an attack block constructed much larger.
 
 
-### REQ-4-1 (set EB to minimum of 8MB at fork)
+### REQ-4-1 (require EB configured to least 8MB at startup)
```
at least

ftrader replied:
thanks, fixed

gandrewstone commented:
we need the ability to change the EB at the point of the fork.  I think that plenty of users might want EB1 until the fork and then EB8.

This is not hard.  We can create CTweaks:
fork.time
fork.excessiveBlock
fork.maxGenerate

I will create a PR that suggests this in the design.

ftrader commented:
> I think that plenty of users might want EB1 until the fork and then EB8.

Agreed with the above - esp. for miners using BU up to now.

I like the idea of these tweaks, only want to point out that discussion had moved in the direction of setting actual MG to the new EB, i.e. to fork.excessiveBlock . But I don't see much harm in allowing a fork.maxGenerate - however, it must be > 1MB otherwise it violates the fork block requirement.

Adding link to basic parameter task for implementation, might be good to add your suggested parameter names there:

https://github.com/ftrader-bitcoinunlimited/BitcoinUnlimited/issues/3

ftrader commented:
@sickpig : changed to "fork EB" as per Slack meeting actions, please review and merge if ok.

I updated the rationale and removed the note as explained in the commit message of c17d2301d32aecf2651ef2e995de14ee6a1353b4 .
